### DISCUSSION TITLE
Container setting

### DISCUSSION THREAD

**Discussion by AFK-TankyMOFO** [FakeFishGames/Barotrauma#8395] [Ideas and suggestions]
This is mainly for single player, and to make AI helpful in inventory management.

Using shift+MMB on a container will gives an option to assign that container specific designation, such as Medical, Weaponry, Ammunition, etc.

AIs with Clean Up task will sort out items accordingly.

And maybe other options like: "This container must have at least/up to a certain number of certain item", this way you can make it so that every "emergency supplies cabinet have at least 3 diving mask and up to 8 bandage, 8 blood packs, and 1 Calyxanide.

They should also put their empty oxygen tanks or batteries in appropriate containers, so that the person with refill task can do their job.

**Comment by Dan-Treacher** [comment]
It might also be nice to have bots by default group stackable items together when there are instances of them inside storage container within lockers. The situation is currently:

- Bot is instructed to 'clear up'
- Bot puts a stackable item (e.g. piece of steel) into a locker that already contains that item within a storage container
- Placed item stays separate from the instances of that item already in the storage container within the locker

I think it'd be nice if the situation became:

- Bot is instructed to 'clear up'
- Bot puts a stackable item (e.g. piece of steel) into a locker that already contains that item within a storage container
- Placed item is moved into the storage container along with the rest of that item

**Comment by gerakis** [comment]
Bots already seem to group items together. So if they find something on the ground like a harpoon, they will put it in the cabinet they holds other harpoons. If there isnt one they seem to just stick it wherever.
